core/math/basis.h

```diff
@@ -238,10 +238,8 @@ struct _NO_DISCARD_ Basis {
 	Basis(const Vector3 &p_axis, real_t p_angle, const Vector3 &p_scale) { set_axis_angle_scale(p_axis, p_angle, p_scale); }
 	static Basis from_scale(const Vector3 &p_scale);
 
-	_FORCE_INLINE_ Basis(const Vector3 &row0, const Vector3 &row1, const Vector3 &row2) {
-		rows[0] = row0;
-		rows[1] = row1;
-		rows[2] = row2;
+	_FORCE_INLINE_ Basis(const Vector3 &p_basis_x, const Vector3 &p_basis_y, const Vector3 &p_basis_z) {
+		set_columns(p_basis_x, p_basis_y, p_basis_z);
```

Comment:
```suggestion
	_FORCE_INLINE_ Basis(const Vector3 &p_x_axis, const Vector3 &p_y_axis, const Vector3 &p_z_axis) {
		set_columns(p_x_axis, p_y_axis, p_z_axis);
```

Reply:
Done. "basis vector" is a stricter term in this context than "axis vector" which is a little bit ambiguous. But I suppose it isn't a big deal and "axis" has already been used everywhere in the codebase.